src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -279,6 +279,10 @@ export class EKSClusterSettingsComponent
       this._externalClusterService.getEKSSecurityGroups(vpc),
     ]).subscribe(([subnets, securityGroups]) => {
       this.subnets = subnets;
+      if (subnets.length) {
```

Comment:
This change is added in this PR as was asked by @imharshita to select default values from 2 different AZ.  So if user wants to go with default values based on VPC change user can create with same selection without user interaction.

cc: @Waseem826